encodedcc.py

```diff
@@ -607,43 +635,33 @@ def patch_set(args, connection):
                 accession = d[i]
                 temp_data.pop(i)
         if not accession:
-            print("No identifier found in headers!  Cannot PATCH data")
+            print("No identifier found in headers! Cannot PATCH data")
             sys.exit(1)
         accession = quote(accession)
         full_data = get_ENCODE(accession, connection, frame="edit")
         if args.remove:
             put_dict = full_data
+            print("OBJECT:", accession)
             for key in temp_data.keys():
                 k = key.split(":")
                 name = k[0]
                 if name not in full_data.keys():
                     print("Cannot PATCH '{}' may be a calculated property".format(name))
                     sys.exit(1)
-                print("OBJECT:", accession)
                 if len(k) > 1:
                     if k[1] in ["list", "array"]:
                         old_list = full_data[name]
                         l = temp_data[key].strip("[]").split(",")
```

Comment:
"l" is a bad name for a python variable; it can clash with list command in python debugger.  Also "b", "s", "n".
Not a big deal though..

Reply:
Agree. Inherited code. Also came across an example of _i_ getting used when not an index variable.